docs/pages/experiments/docs/installation.md

```diff
@@ -0,0 +1,37 @@
+# Installation
+
+<p class="description">Installation component.</p>
+
+## Tabs
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install @mui/material @emotion/react @emotion/styled
+```
+
+```bash yarn
+yarn add @mui/material @emotion/react @emotion/styled
+```
+
+</codeblock>
+
+<codeblock storageKey="license">
+
+```bash MIT
```

Comment:
We almost never use ` ```bash ` in the codebase (I can only find a few Sebastian used). Would it be more coherent with
````suggestion
```sh MIT
````
?

We could replace the other few instances as well.

Reply:
As Andrew noticed:

> Currently, `sh` is using `prism-jsx`. Should it now use `prism-bash` instead?

I assume it's a kind quick fix that get done: "I need to put some shell code, let use `sh` and we will say `sh` renders `prism-jsx`" Now that we import `prism-bash` I think it would make more sense to use `bash` everywhere to respect the `prism` naming

Reply:
sh and bash seems to be mostly the same https://stackoverflow.com/questions/5725296/difference-between-sh-and-bash so I think that it would be great to throw an error if we use the language we decided not to normalize on